pkg/storage/unified/apistore/restoptions.go

```diff
@@ -8,6 +8,7 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
```

Comment:
it doesn't seem like this should pass the depguard check 🤔

Reply:
I have created https://github.com/grafana/grafana/issues/94724